docs/source/index.rst

```diff
@@ -1,4 +1,50 @@
-Welcome to SyReC's documentation!
-=================================
```

Comment:
Why the change here? A welcome certainly deserves an exclamation mark ;o)

Reply:
done 👍🏻